apollo-backend/src/test/java/io/logz/apollo/helpers/ModelsGenerator.java

```diff
@@ -180,7 +182,15 @@ public static Deployment createAndSubmitDeployment(ApolloTestClient apolloTestCl
 
         // Now we have enough to create a deployment
         Deployment testDeployment = ModelsGenerator.createDeployment(testService, testEnvironment, testDeployableVersion);
-        testDeployment.setId(apolloTestClient.addDeployment(testDeployment).getId());
+        MultiDeploymentResponseObject result = apolloTestClient.addDeployment(testDeployment);
+
+        try {
+            LinkedHashMap deployment = (LinkedHashMap) result.getSuccessful().get(0).get("deployment");
```

Comment:
You are using the java client, please have specific getters all around.. you dont need to know the json structure from within